backend/src/bundles/strava/constants/constants.ts

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line simple-import-sort/exports
```

Comment:
Is this due to a circular dependency, or is there another reason?
Would importing STRAVA_URL directly from the file in STRAVA_API_URL instead of from constants be a helpful solution?